.github/workflows/ci.yaml

```diff
@@ -65,10 +57,9 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v1
       - name: Create kind cluster
-        uses: helm/kind-action@master
+        uses: monotek/kind-action@kind070
```

Comment:
Isn't there a chance we get this in the "helm" action any time soon?

Reply:
We can wait before merging this so if its merged there i change it back here.

Reply:
My understanding is that without this change, the CI pipeline doesn't work for Hono and hawkBit, right? If so, then I would rather have this change merged now and then later, once it has made its way into the *official* `helm/kind-action`, change it back. That way we would be able to make some progress with the Hono and hawkBit charts ...

Reply:
I was hoping for a quick merge upstream. But yes, let's merge that and make progress with our stuff.